src/Stratis.Bitcoin/Consensus/ConsensusManager.cs

```diff
@@ -765,7 +762,7 @@ private void HandleMissingBlocksGap(ChainedHeader newTip)
                 throw new ConsensusException("Block must be partially or fully validated.");
             }
 
-                var validationContext = new ValidationContext() { Block = blockToConnect.Block, ChainTipToExtand = chainTipToExtand};
+                var validationContext = new ValidationContext() { Block = blockToConnect.Block, ChainTipToExtand = blockToConnect.ChainedHeader };
```

Comment:
extand->extend?

Reply:
I think we should rename this from `ChainTipToExtand` to `BlockToConnect`